### PR TITLE
#11332: Move `ttnn/examples` `ttnn/ttnn/examples` so we can enable directly calling them for users, but not meant to be part of ttnn API

### PR DESCRIPTION
**Pull request by tt-rkim**
### Ticket

#11332 

### Problem description

a third-party really wants a clearer, more succinct example to test that the installation of the software works. This example is meant to be installation-method independent.

### What's changed

- Moved `ttnn/examples` to `ttnn/ttnn/examples`, meaning they're now callable as command-line modules
- These examples are still not meant to be a usable API for users, rather just things they can quickly run to verify installation
- Added the specific test to wheel tests

### Checklist
- [x] Post commit CI passes
- [x] Blackhole Post commit (if applicable)
- [x] Model regression CI testing passes (if applicable)
- [x] New/Existing tests provide coverage for changes

### CODE REVIEW

**Review by ayerofieiev-tt — APPROVED** [review]
@tt-rkim do we run all these examples on CI?

**Comment by tt-rkim** [comment]
@ayerofieiev-tt The working ones yes - which I think should be all of them: `tests/scripts/run_ttnn_examples.sh`

The modified example in here is the only one that runs in ttnn post commit + wheel tests. Other examples only run in ttnn post commit.

**Comment by tt-rkim** [comment]
Waiting on @zzigler-tt and his confirmation.

**Comment by tt-rkim** [comment]
passing post commit: https://github.com/tenstorrent/tt-metal/actions/runs/10457654746/job/28958421614

File: docs/source/ttnn/ttnn/usage.rst
Comment (TT-billteng) [inline docs/source/ttnn/ttnn/usage.rst:1]: How would the user get to here from the front page of our GH repo?

**Review by TT-billteng — APPROVED** [review]
Thanks @tt-rkim !

**Review by ttmchiou — APPROVED** [review]

**Comment by tt-rkim** [comment]
@zzigler-tt Any update on the priority of this work / changes in requirements? Recall we didn't agree on specifics very much, so I used my own interpretation of the ask.

**Comment by zzigler-tt** [comment]
@tt-rkim The requirements are the same, provide a link to a test case for users to validate the install is successful. @TT-billteng this should be very straightforward based on the chat with @jvasilje and me. If you have any further questions let's chat.

**Pull request merged**